### PR TITLE
Bugfix/#2535/create admin page sidebar

### PR DESCRIPTION
**Pull request by AlekOK**
I created a new page and added sidebar.

### CODE REVIEW

**Review by paslavskyi9roman — APPROVED** [review]

**Review by Edw1x — APPROVED** [review]

**Review by idzhersh — APPROVED** [review]

File: src/app/ubs-admin/components/ubs-admin-table/ubs-admin-table.component.html [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+<div class="btn-group">
+  <button type="button" class="btn btn-default" class="amount-items"><i class="fa fa-angle-down" aria-hidden="true"></i> 10</button>
+  <button type="button" class="btn btn-default" class="filter-button">
+    <div><mat-icon>filter_list</mat-icon><span>Фільтрувати</span></div>
+  </button>
+  <button type="button" class="btn btn-default" class="view-table-btn" data-toggle="dropdown">
+    <mat-icon>create</mat-icon>Вид таблиці
+  </button>
+  <ul class="dropdown-menu">
+    <li>Інформація про замовлення</li>
```
Comment (JimHolman1199) [inline src/app/ubs-admin/components/ubs-admin-table/ubs-admin-table.component.html:14]: Move all text to separate file in the asset folder.

File: src/app/ubs-admin/components/ubs-admin-table/ubs-admin-table.component.html [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+    <mat-header-row *matHeaderRowDef="displayedColumns"></mat-header-row>
+    <mat-row *matRowDef="let row; columns: displayedColumns"></mat-row>
+  </table>
+</div>
+<div>
+  <mat-paginator [pageSizeOptions]="[2, 5, 10]" showFirstLastButtons></mat-paginator>
+</div>
+<ng-template #spinner>
+  <!-- TODO: use our Greenciry spinner -->
+  <!-- <mat-spinner></mat-spinner> -->
```
Comment (JimHolman1199) [inline src/app/ubs-admin/components/ubs-admin-table/ubs-admin-table.component.html:96]: it should be another spinner
the one we have in a shared folder
write me, so i can guide you about this

File: src/app/ubs-admin/components/ubs-admin-table/ubs-admin-table.component.scss [hunk 3]
```diff
@@ -0,0 +1,256 @@
...
+  justify-content: center;
+  font-size: 14px;
+  width: 171px;
+  min-height: 39px !important;
+  min-width: 48px !important;
+  max-width: 300px !important;
+  overflow: hidden !important;
+  word-wrap: break-word !important;
+  white-space: nowrap !important;
+  overflow-wrap: break-word !important;
```
Comment (JimHolman1199) [inline src/app/ubs-admin/components/ubs-admin-table/ubs-admin-table.component.scss:38]: https://css-tricks.com/when-using-important-is-the-right-choice/
well, this looks not like best practices

File: src/app/ubs-admin/components/ubs-admin-table/ubs-admin-table.component.html (hunk 1 above)
Comment (AlekOK) [reply src/app/ubs-admin/components/ubs-admin-table/ubs-admin-table.component.html:14]: skip

File: src/app/ubs-admin/components/ubs-admin-table/ubs-admin-table.component.html (hunk 2 above)
Comment (AlekOK) [reply src/app/ubs-admin/components/ubs-admin-table/ubs-admin-table.component.html:96]: Created shared module and added spinner component for all modules

**Review by SofiiaBaziv — APPROVED** [review]

**Pull request merged**